package.json

```
@@ -2,65 +2,78 @@
   "name": "fast-chunk-string",
   "version": "1.0.1",
   "description": "Chunk string into equal substrings with unicode support",
-  "license": "MIT",
+  "keywords": [
+    "string",
+    "split",
+    "chunk",
+    "unicode",
+    "utf",
+    "utf8",
+    "substring"
+  ],
   "repository": "vladgolubev/fast-chunk-string",
+  "license": "MIT",
   "author": {
     "name": "Vlad Holubiev",
     "email": "golubev.vld@gmail.com",
     "url": "vladholubiev.com"
   },
-  "engines": {
-    "node": ">=6"
-  },
+  "files": [
+    "src/index.ts"
```

```suggestion
    "lib"
```

`files` is what's getting published with the npm package